.gitignore

```diff
@@ -4,4 +4,6 @@ node_modules
 dist/
 mochawesome-report/
 .vscode/
-yarn-error.log
\ No newline at end of file
+yarn-error.log
+output
+steps.d.ts
```

Comment:
I didn't see any steps.d.ts file , why do we need to ignore?

Reply:
The type definition file is to allow for intellisense.  Autocompletion as codecept puts it.
It's just to make writing  tests less tricky for me.
This is how Codecept advised to get code completion.